src/wp-content/themes/twentytwentyfour/readme.txt

```diff
@@ -16,7 +16,7 @@ Twenty Twenty-Four is designed to be flexible, versatile and applicable to any w
 = 1.0 =
 * Released: November 7, 2023
 
-https://wordpress.org/support/article/twenty-twenty-four-changelog#Version_1.0
+https://wordpress.org/documentation/article/twenty-twenty-four-changelog/#Version_1.0
```

Comment:
If this is changed to "Initial release" until version 1.1, the changelog article would not need to be published immediately on the release date.

Reply:
https://github.com/WordPress/twentytwentyfour/pull/718 merged this change to the `documentation` URL.

Reply:
I removed the change from this PR because it is already included in https://github.com/WordPress/wordpress-develop/pull/5623.